requirements.txt

```diff
@@ -43,3 +43,4 @@ XlsxWriter==0.9.3
 xlwt==1.3.*
 xlrd==1.0.0
 pypiwin32 ; sys_platform == 'win32'
+firebase-admin==2.17.0
```

Comment:
@d-fence

Reply:
@mreficent It's already being taken care of, thanks